Hardware/Hardware/CardReader/CardPresentDetails.swift

```diff
@@ -29,3 +29,17 @@ public struct CardPresentDetails {
     /// (Only applicable to EMV payments) The authorization data from the card issuer.
     public let emvAuthData: String?
 }
+
+extension CardPresentDetails: Equatable {
```

Comment:
I should have thought of this before, but `CardPresentTransactionDetails` might be a clearer name for this struct. I had to think about it for a few seconds (and look at the heading comment again) to remember what this struct's responsibility was.

Reply:
Makes sense, I'll rename now